### PR TITLE
snmp-extend: ups-apcups

### PR DESCRIPTION
**Pull request by crcro**
shell script to export linux apc ups app status

### CODE REVIEW

File: snmp/ups-apcups.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+################################################################
+# copy this script to somewhere like /opt and make chmod +x it #
+# edit your snmpd.conf and include                             #
+# extend ups-apcups /opt/ups-apcups.sh                         #
```
Comment (laf) [inline snmp/ups-apcups.sh]: We need to standardise the install directory - people can change it but we should tell those that don't know to / want to. I think we have suggestions in the docs of where to store these, use one of those.

Comment (crcro) [inline snmp/ups-apcups.sh]: The suggestion from docs is for agent setup, maybe we should put this straight now, and I'll change the docs and all other. I suggest something like /opt/librenms/apps or /etc/snmpd/extend

Comment (laf) [inline snmp/ups-apcups.sh]: /etc/snmp/ seems to be used regularly so I'd go with that - @murrant ?

Comment (murrant) [inline snmp/ups-apcups.sh]: That is why I used that in my documentation entries, so I would prefer /etc/snmp

**Comment by laf** [comment]
Just needs the header changing of where to install this to match docs then good to go.

**Comment by crcro** [comment]
@laf .. done.

**Pull request merged**